Update JetBrains plugin page (#3366)
dart-lang/site-www: src/tools/jetbrains-plugin.md

this page has resources to help you get started quickly
and find more information when you need it.

{{site.alert.note}}
  [WebStorm,](https://www.jetbrains.com/webstorm/)
  a JetBrains IDE for client-side development,

>   comes with the Dart plugin pre-installed.  

Reviewer: ```suggestion
  comes with the Dart plugin pre-installed. 
```

> Install a JetBrains IDE if you don't already have one.

Reviewer: I think we need to be clearer that this is an OR choice. Maybe:

```suggestion
Install a JetBrains IDE if you don't already have one. Choose one:
```

> * [Download IntelliJ IDEA.][IDEA]{:target="_blank" rel="noopener"}
> * To try out the latest Dart language features and IntelliJ functionality,
>   [install IntelliJ IDEA EAP.][IDEA EAP]{:target="_blank" rel="noopener"}
> * [Find another Jetbrains product.][Other]{:target="_blank" rel="noopener"}

Reviewer: ```suggestion
* [IntelliJ IDEA][IDEA]{:target="_blank" rel="noopener"}
* [IntelliJ IDEA EAP][IDEA EAP]{:target="_blank" rel="noopener"}
  (for early access to the latest Dart language features and IntelliJ functionality)
* [Another JetBrains product][Other]{:target="_blank" rel="noopener"}
```

Author: I wasn't a fan of the periods, so thanks for this!

Reviewer: > I wasn't a fan of the periods, so thanks for this!

:) Unlike some other companies, we use periods for list items that are complete sentences (but not generally for other list items). I think we used to use periods for more kinds of list items, but that's just silly. https://developers.google.com/style/lists#capitalization-and-end-punctuation
